index.js

```diff
@@ -23,3 +24,174 @@ navbarLinks.addEventListener('click', () => {
+      <p class="recent-work-paragraph">
+      ${project.projectDescription}
+      </p>
+      <ul
+        class="recent-work-ul display-flex justify-content-space-between"
+      >
+      ${callTechList(project.projectTechnologies)}      
+      </ul>
+      <button
+       projectData-key="${project.key}"
+       id="${project.id}"
+        type="button"
+        aria-label="see project"
+        class="see-project-button border-none see-project-modal"
+      >
+        see projects
+      </button>
+    </article>
+   </div>`;
+  }
+  return innerHTML;
+}
+
+const projectSkeleton = implementProject();
+workSection.innerHTML = projectSkeleton;
+function seeProectModal(projects) {
+  const populateProjectModal = (project) => `
+    <ul class="display-flex justify-content-space-between align-items">
+      <li><h3>${project.projectName}</h3></li>
+      <li>
+         <span class="close-modal-button"> &times;</span> 
+      </li>
+    </ul>
+    <div>
+      <img
+        src=${project.projectImage}
+        alt="Modal image"
+        class="recent-work-demo-modal"
+      />
+    </div>
+    <p class="about-project-paragraph text-align margin-0-auto">
+    ${project.projectFullDescription}
+    </p>
+
+    <ul class="recent-work-ul-modal display-flex justify-content-center">
+     ${callTechList(project.projectTechnologies)}
+    </ul>
+
+    <div
+      class="display-flex flex-d-col flex-d-row-lg margin-0-auto modal-button-div"
+    >
+      <button
+        type="submit"
+        aria-label="see project"
+        class="see-project-button border-none display-flex modal-container-button align-items justify-content-space-between"
+      >
+        <span> See Live </span>
+        <span>
+          <img src="./images/see-live-icon.png" alt="see live link image"
+        /></span>
+      </button>
+
+      <button
+        type="submit"
+        aria-label="see project"
+        class="see-project-button border-none display-flex modal-container-button align-items justify-content-space-between"
+      >
+        <span> See Source </span>
+        <span>
+          <img src="./images/github-icon.png" alt="github link image"
+        /></span>
+      </button>
+    </div>
+    `;
+  const handleSeeModal = (event) => {
+    const currentId = event.target.id;
+    const currentProject = projects.find((project) => project.id === currentId);
+    const modalElement = document.querySelector('.modal-container');
+    modalElement.innerHTML = populateProjectModal(currentProject);
+    modalElement.classList.remove('display-none-sm');
+    const closeModal = modalElement.querySelector('.close-modal-button');
+
+    closeModal.addEventListener('click', () => {
+      modalElement.classList.add('display-none-sm');
+    });
+  };
+
+  const allSeeProectButtons = document.querySelectorAll('.see-project-modal');
+  allSeeProectButtons.forEach((button) => {
+    button.addEventListener('click', handleSeeModal);
+  });
+}
+seeProectModal(projectData);

```

Comment:
- [ ] You did well implementing the modal with the dynamic informations from the project objects, but the styling needs some improvement. First of all, the text is unreadable because the modal has a transparent background. As you can see on the Figma template, the section behind the modal is semi-transparent, with a blur effect added so the text can actually be readable, you can play around with opacity and filters to achieve that effect, I suggest you take care of this issue 👍
- [ ] Please also make sure to get as close to the design as possible by matching margins, paddings etc. Currently, all your elements have no spacing and are aligned at the top.

| Figma modal  | Your modal |
| ------------- | ------------- |
| ![](https://i.imgur.com/lJVlKYb.png)  | ![](https://i.imgur.com/lFDFQSK.png)  |

Reply:
Thanks for the review change made